modules/llm-cache/tests/refcnt_map_test.cc

```diff
@@ -203,6 +203,19 @@ Status checkRefCnt(std::string ipc_socket) {
     }
     refcnt_map->IncSetRefcnt(blockIDSetToAdd);
   }
+
+  ObjectID globalCacheObjectID;
+  blockIDSetToAdd.clear();
+  VINEYARD_CHECK_OK(client[0].GetName(llmCacheObjectName, globalCacheObjectID));
+  std::shared_ptr<KVStateCache> kvStateCache =
+      std::dynamic_pointer_cast<KVStateCache>(
+          client[0].FetchAndGetObject(globalCacheObjectID));
+  kvStateCache->GetCurrentBlockIDSet(blockIDSetToAdd);
+  refcnt_map->IncSetRefcnt(blockIDSetToAdd);
+  if (kvStateCache->id() != globalCacheObjectID) {
+    client[0].DelData(kvStateCache->id());
```

Comment:
It might failure...

Keep in mind that any interaction with vineyard may fail, and requires status checking.